Data Polish-Backend/Data-Polish-Backend/src/main/java/com/example/Data/PolishBackend/Service/FileDownloadService.java

```diff
@@ -0,0 +1,33 @@
+package com.example.Data.PolishBackend.Service;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.stereotype.Service;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RequestParam;
+
+@Service
+public class FileDownloadService {
+    @Autowired
+    private JdbcTemplate jdbcTemplate;
+
+    public ResponseEntity<String> getFileDetails(String jobID) {
+        String sql = "SELECT rawurl, dataprofileoutput, datacleaningoutput FROM jobsandblobs WHERE jobid = ?";
+        try {
+            // Execute the query to retrieve the file details
+            String fileDetails = jdbcTemplate.queryForObject(sql, (rs, rowNum) -> {
+                String rawurl = rs.getString("rawurl");
+                String dataprofileoutput = rs.getString("dataprofileoutput");
+                String datacleaningoutput = rs.getString("datacleaningoutput");
+                return "{\"rawurl\":\"" + rawurl + "\", \"dataprofileoutput\":\"" + dataprofileoutput + "\", \"datacleaningoutput\":\"" + datacleaningoutput + "\"}";
```

Comment:
can you use string builder in here

Reply:
will do that in later version when we enhance the quality of code